package.json

```diff
@@ -48,7 +48,9 @@
     "test:functional": "echo \"We don't have functional tests yet\"",
     "test": "npm run lint && npm run test:unit && npm run test:examples && npm run test:functional",
     "examples": "NODE_ENV=test shell-exec 'npm run start' 'babel-node ./scripts/launchBrowser http://localhost:8080/examples'",
-    "semantic-release": "semantic-release pre && npm publish && semantic-release post"
+    "semantic-release": "semantic-release pre && npm publish && semantic-release post",
+    "selenium": "selenium-standalone start",
```

Comment:
I'd dockerize the whole visual regression test, because same way you can argue about installing graphicsmagick